platform/src/test/java/org/exoplatform/platform/qa/ui/platform/forum/ForumTopicLockTestIT.java

```diff
@@ -0,0 +1,96 @@
+package org.exoplatform.platform.qa.ui.platform.forum;
+
+import com.codeborne.selenide.Condition;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.forum.pageobject.ForumCategoryManagement;
+import org.exoplatform.platform.qa.ui.forum.pageobject.ForumForumManagement;
+import org.exoplatform.platform.qa.ui.forum.pageobject.ForumHomePage;
+import org.exoplatform.platform.qa.ui.forum.pageobject.ForumTopicManagement;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.core.PLFData.password;
+import static org.exoplatform.platform.qa.ui.core.PLFData.username;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.forum.ForumLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+import static org.exoplatform.platform.qa.ui.selenium.testbase.LocatorTestBase.ELEMENT_SKIP_BUTTON;
+
+/**
+ * Created by esmaoui on 16/03/2018.
+ */
+
+
+@Tag("sniff")
+@Tag("forum")
+public class ForumTopicLockTestIT extends Base {
```

Comment:
You can add this TC in platform/forum/ForumTopicTestIT. Should not create a class for each TC.